doc/reference/cmdline.rst

```diff
@@ -6,282 +6,297 @@
 Command-line interface: odoo.py
 ===============================
 
-.. _reference/cmdline/server:
+.. _reference/cmdline/env:
 
-Running the server
-==================
+Installing Development Environment
+==================================
 
-.. program:: odoo.py
+    .. note:: This section applies to those developing within
```

Comment:
Hah, where's that GIF of panda smashing the keyboard? (Something stupid I did at the last second, after a long day. I knew better too...) Will fix...